src/App.tsx

```diff
@@ -1,61 +1,123 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './Types/User';
+import { Todo } from './Types/Todo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [currentTodos, setCurrentTodos] = useState(todos);
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [title, setTitle] = useState('');
+  const [isTitled, setIsTitled] = useState(true);
+  const [userIsSelected, setUserIsSelected] = useState(true);
+
+  const getMaxID = Math.max(...currentTodos.map((todo: Todo) => todo.id));
```

Comment:
here you saved max ID, so the name of variable can be just 'maxId' or 'maxUserId'

Reply:
agree
```suggestion
  const maxID = Math.max(...currentTodos.map((todo: Todo) => todo.id));
```